pkg/network/proxy/hybridproxier.go

```diff
@@ -115,15 +115,15 @@ func NewHybridProxier(
 }
 
 func (proxier *HybridProxier) OnNodeAdd(node *corev1.Node) {
-	// TODO implement https://github.com/kubernetes/enhancements/pull/640
+	proxier.mainProxy.OnNodeAdd(node)
```

Comment:
iptables proxiers calls https://github.com/kyrtapz/sdn/blob/f50597184684578f109dc685198f2ad22aa0ec13/vendor/k8s.io/kubernetes/pkg/proxy/iptables/proxier.go#L652, is this an issue?